source/utils/bpls/bpls.h

```diff
@@ -34,34 +35,12 @@ namespace utils
 #define MAX_MASKS 10
 #define MAX_BUFFERSIZE (10 * 1024 * 1024)
 
-/* global defines needed for the type creation/setup functions */
-enum ADIOS_DATATYPES
-{
-    adios_unknown = -1,
-    adios_byte = 0,
-    adios_short = 1,
-    adios_integer = 2,
-    adios_long = 4,
-    adios_unsigned_byte = 50,
-    adios_unsigned_short = 51,
-    adios_unsigned_integer = 52,
-    adios_unsigned_long = 54,
-    adios_real = 5,
-    adios_double = 6,
-    adios_long_double = 7,
-    adios_string = 9,
-    adios_complex = 10,
-    adios_double_complex = 11,
-    adios_string_array = 12,
-    adios_long_double_complex = 13
-};
```

Comment:
@chuckatkins do you know why these had assigned values?

Reply:
They were initially put in thinking we'd implement them but it didn't really happen.  @KyleFromKitware has the long double support on his list but there's some other things to get to first so it will be a while.  Safe to remove.

Reply:
These are the actual byte values of encoding these types in the BP file format. 
I think I just used for something in bpls or wanted to use it. I wouldn't say it is safe to remove, there are some functions in bpls to translate between adios types and adios types (sic!)

Reply:
My mistake then.  Leave them.

Reply:
Currently this PR switches bpls from `enum ADIOS_DATATYPES` to using the new `adios2::Type` enumeration directly, so `type_to_enum` is no longer needed.  Unless `bpls` uses the actual integer values that `ADIOS_DATATYPES` had for something, why shouldn't they be removed?

Or, should I restore `enum ADIOS_DATATYPES` as it was and make `type_to_enum` convert between the two different enumerations?

Reply:
Okay, sorry for the flame. You cleaned up bpls nicely with the new Type class. So it's okay to remove. The only places where these values count in adios2 are the engine serializers and deserializers. 

I just added this enum to bpls when adding the old bpls C program to ADIOS2 with the least amount of work (a signature of my attitude). Thanks for cleaning this up.

Reply:
Great, thanks for explaining!